nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -62,6 +62,12 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [hebbot](https://github.com/haecker-felix/hebbot), a Matrix bot to generate "This Week in X" like blog posts. Available as [services.hebbot](#opt-services.hebbot.enable).
 
+- [Python Matter
+  Server](https://github.com/home-assistant-libs/python-matter-server), a
+  Matter Controller Server exposing websocket connections for use with other
+  services, notably Home Assistant. Available as
+  [services.matter-server](#opt-services.matter-server.enable)
```

Comment:
```suggestion
- [Python Matter Server](https://github.com/home-assistant-libs/python-matter-server), a
  Matter Controller Server exposing websocket connections for use with other services, notably Home Assistant.
  Available as [services.matter-server](#opt-services.matter-server.enable)
```
keeping links in one line makes them easy searchable. Also we don#t need to adhear to such short lines. They usually just make things complicated. Doing one sentence per line is a pretty good alternative.